pkgs/development/tools/misc/premake/5.nix

```diff
@@ -10,49 +10,58 @@
 
 stdenv.mkDerivation rec {
   pname = "premake5";
-  version = "5.0.0-beta4";
+  version = "5.0.0-beta6";
 
   src = fetchFromGitHub {
     owner = "premake";
     repo = "premake-core";
     rev = "v${version}";
-    sha256 = "sha256-sNLCyIHWDW/8jIrMFCZAqtWsh4SRugqtPR4HaoW/Vzk=";
+    sha256 = "sha256-wMy9uqtlWAB+o/D169CLNQDd80zhsWKyNBXzKxhH7n4=";
   };
 
   buildInputs =
-    [ libuuid ]
+    [
+      libuuid
+    ]
     ++ lib.optionals stdenv.hostPlatform.isDarwin [
       Foundation
       readline
     ];
 
-  patches = [ ./no-curl-ca.patch ];
+  # patches = [ ./no-curl-ca.patch ];
   postPatch =
     ''
       substituteInPlace contrib/curl/premake5.lua \
-        --replace "ca = nil" "ca = '${cacert}/etc/ssl/certs/ca-bundle.crt'"
+        --replace-fail "ca = nil" "ca = '${cacert}/etc/ssl/certs/ca-bundle.crt'"
+      substituteInPlace premake5.lua \
+        --replace-fail "arch=ARM64" }" "arch=ARM64" or arch=arm64}"
+      substituteInPlace premake5.lua \
+        --replace-fail "{ \"ARM64\", \"ARM64\" }", "{ \"arm64\", \"ARM64\" }",
     ''
     + lib.optionalString stdenv.hostPlatform.isDarwin ''
       substituteInPlace premake5.lua \
-        --replace -mmacosx-version-min=10.4 -mmacosx-version-min=10.5 \
         --replace-fail '"-arch arm64"' '""' \
         --replace-fail '"-arch x86_64"' '""'
     ''
     + lib.optionalString stdenv.hostPlatform.isStatic ''
       substituteInPlace \
         binmodules/example/premake5.lua \
         binmodules/luasocket/premake5.lua \
-        --replace SharedLib StaticLib
+        --replace-warn SharedLib StaticLib
```

Comment:
maybe -fail too? Or is it that you are not sure since you can't build isStatic?

Reply:
-fail seems like the right thing to do. Fixed.

Reply:
Wasn't it -fail in the previous iteration? And perhaps -quiet above should also be -fail?

Reply:
It was in the -beta6 source. Give me a moment and I'll update these.